index.js

```diff
@@ -28,13 +28,14 @@ module.exports = {
     var self = this;
     var app = config.app;
     var options = config.options;
+    var baseURL = options.liveReloadBaseUrl || options.baseURL;
```

Comment:
Can you switch this to `options.liveReloadBaseUrl || options.rootURL || options.baseURL`?